doc/README

```diff
@@ -0,0 +1,14 @@
+===========================================
+NISTATS: functional MRI analysis in python
+===========================================
+
+This repository contains code and documentation to perform fMRI analysis. The requirements are ...
```

Comment:
yes, now we have the barebones to update examples and doc references as we add the other PRs. I would say a good tutorial is missing before publication in the website